packages/realm/src/app-services/SyncConfiguration.ts

```diff
@@ -148,20 +182,31 @@ export type SyncConfiguration = FlexibleSyncConfiguration | PartitionSyncConfigu
 
 /** @internal */
 export function toBindingSyncConfig(config: SyncConfiguration): binding.SyncConfig_Relaxed {
-  const { user, flexible, partitionValue, onError, _sessionStopPolicy, customHttpHeaders, clientReset } = config;
+  const { user, flexible, partitionValue, onError, _sessionStopPolicy, customHttpHeaders, ssl, clientReset } = config;
 
   return {
     user: user.internal,
     partitionValue: flexible ? undefined : EJSON.stringify(partitionValue),
+    flxSyncRequested: !!flexible,
     stopPolicy: _sessionStopPolicy
       ? toBindingStopPolicy(_sessionStopPolicy)
       : binding.SyncSessionStopPolicy.AfterChangesUploaded,
     customHttpHeaders,
-    flxSyncRequested: !!flexible,
+    clientValidateSsl: ssl?.validate,
+    sslTrustCertificatePath: ssl?.certificatePath,
+    sslVerifyCallback: ssl?.validateCertificates
+      ? binding.Helpers.makeSslVerifyCallback(toSSLVerifyCallbackWithListArguments(ssl.validateCertificates))
+      : undefined,
     ...parseClientResetConfig(clientReset, onError),
   };
 }
 
+/** @internal */
```

Comment:
On second thought, don't think this annotation should be needed since the function is not exported. However, there are several non-exported functions in this file using this annotation.